packages/edit-site/src/components/editor/index.js

```diff
@@ -17,17 +19,32 @@ import Sidebar from '../sidebar';
 import BlockEditor from '../block-editor';
 
 function Editor( { settings } ) {
-	return (
+	const template = useSelect(
+		( select ) =>
+			select( 'core' ).getEntityRecord(
+				'postType',
+				'wp_template',
+				settings.templateId
+			),
+		[]
+	);
```

Comment:
just noting that the logic in this PR is already available in editPost, we can tell it to just load the template directly. so for now, these packages are still equivalent.

Reply:
They are very different.

This doesn't load `@wordpress/editor` and all of its unnecessary-for-this-context logic.

Reply:
Ultimately, it might have to load the editor package because there's a lot of useful components there. Save buttons,  Editing Post Title (template title)...

I think you're right when you say,  they'll be different it's just not there yet.

Reply:
The package, yes, but not the provider and/or data flow.

Reply:
The package, yes, but not the provider and/or data flow.